third_party/move/move-prover/boogie-backend/src/spec_translator.rs

```diff
@@ -991,15 +996,31 @@ impl<'env> SpecTranslator<'env> {
                 self.translate_call(node_id, oper, &[args[args.len() - 1].clone()]);
                 emit!(self.writer, &")".repeat(count));
             },
+            Operation::Abort => {
+                let exp_bv_flag = global_state.get_node_num_oper(node_id) == Bitwise;
+                emit!(
```

Comment:
Can we show this error, too?

Reply:
I think you mean the error below. All operations should already be handled in early stages so it is hard to add a test case to generate this error.